### DISCUSSION TITLE
Sideloader (still a provisional name) Pre-release 2

### DISCUSSION THREAD

**Discussion by Dadoum** [Dadoum/Sideloader#28] [Announcements]
macOS, Windows support, a CLI, a lot of bug fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/Dadoum/Sideloader/releases/tag/1.0-pre2'>Sideloader (still a provisional name) Pre-release 2</a>.</em>

**Comment by dylanchapell** [comment]
I get a segfault running sideloader-gtk-linux-aarch64 and installing the SideStore 0.3.1 ipa
Output of the app:
[sideloadersegfault.txt](https://github.com/Dadoum/Sideloader/files/14855423/sideloadersegfault.txt)

Additional info from GDB: 
```
Thread 45 "sideloader-gtk-" received signal SIG34, Real-time event 34.
[Switching to Thread 0xffffb124e200 (LWP 46339)]
0x0000fffff7c3c914 in __futex_abstimed_wait_cancelable64 () from /lib64/libc.so.6
```

**Reply by Dadoum** [reply]
What is the version of libplist on your system?

**Reply by dylanchapell** [reply]
2.3.0
